backend/tests/integration-async/migrator/test_migrator_api.py

```diff
@@ -335,19 +338,14 @@ def test_continuous_migration_with_invalid_control_number(
 
     cache.rpush("legacy_records", zlib.compress(raw_record_citer))
     cache.rpush("legacy_records", zlib.compress(raw_record_cited))
+    cache.rpush("legacy_records", b"END")
 
-    assert cache.llen("legacy_records") == 2
+    assert cache.llen("legacy_records") == 3
 
     with pytest.raises(ValueError):
         continuous_migration()
 
-    record_citer = InspireRecord.get_record_by_pid_value(citer_control_number, "lit")
-    record_citer_es = InspireSearch.get_record_data_from_es(record_citer)
-    result_citer_control_number = record_citer_es["control_number"]
-
-    assert citer_control_number == result_citer_control_number
```

Comment:
I removed this as the migration of records will be triggered later when encountering the `END` message. I don't think it matters in practice as exceptions at this stage never happen.